src/dyn/dyn_control_params.cpp

```diff
@@ -82,7 +82,7 @@ dyn_control_params::dyn_control_params(){
 
   ///============================ QTAG =============================================
   qtag_pot_approx_method = 0;
-
+  states = vector<int>();

```

Comment:
these are not parameters, these are dynamical variables, so they should not appear in the control_params. Please change